rewrite/main.rs

```diff
@@ -68,6 +108,13 @@ fn main() {
                 .help("Location and name of the output file."),
         )
         .get_matches();
+
+    let mut udfs = Vec::new();
+    if let Some(udf_file) = matches.value_of("udf") {
```

Comment:
Looks like we do not support multiple UDFs yet. Should add a todo here.

Reply:
There's a TODO below, on line 136